packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -631,6 +633,7 @@ const signCoinjoinTx =
                             serialize: false,
                             unlockPath,
                             override: true, // override current call (override SUITE.LOCK)
+                            chunkify: addressDisplay === 'chunked',
```

Comment:
I know this is only draft and not ready for review, just a note - would be nice to replace all 'chunked' strings by enum AddressDisplayOptions.CHUNKED

Reply:
e5bba442ba785d3e9eef8f250b3e69519946cb10